tests/integration_tests/tests/agentless_tests/test_chown.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from cloudify_rest_client.exceptions import CloudifyClientError, ForbiddenError
+
+from integration_tests import AgentlessTestCase
+from integration_tests.tests.utils import get_resource as resource
+
+pytestmark = pytest.mark.group_deployments
+
+
+class BlueprintChownTest(AgentlessTestCase):
+    BLUEPRINT_ID = 'bp'
+    BLUEPRINT_FILENAME = 'empty_blueprint.yaml'
+
+    def test_successful(self):
+        self.client.blueprints.upload(
+            resource('dsl/{}'.format(self.BLUEPRINT_FILENAME)),
+            entity_id=self.BLUEPRINT_ID)
+        self.client.users.create('user1', 'password1', 'default')
+        self.client.blueprints.update(self.BLUEPRINT_ID, {'creator': 'user1'})
+
+        blueprint = self.client.blueprints.get(self.BLUEPRINT_ID)
+        assert blueprint.get('created_by') == 'user1'
+
+        try:
+            self.client.users.delete('user1')
+        except CloudifyClientError as ex:
+            assert ex.status_code == 400
+            assert 'cannot delete user' in str(ex).lower()
+        else:
+            assert False
```

Comment:
probably better written using `assertRaises`? the bare `assert False` will produce misleading exceptions

here's an example, also using `cm.exception` (which you can `str()`): https://github.com/cloudify-cosmo/cloudify-manager/blob/f2ecc3bef310d57284a1e707c19406017100f3a4/tests/integration_tests/tests/agentless_tests/multi_tenancy/test_users.py#L110-L112